### ISSUE TITLE
save config at run time

### ISSUE THREAD

**Issue by Remi-Gau** [cpp-lln-lab/CPP_PTB#180]
currently all the config details are not saved by CPP_BIDS anymore because putting all metada in `_bold.json` makes fmriprep and mriqc freak out.

but we now don't have any trace of the experimental parameters used for a given run, which is really not ideal for reproducibility (or methods writing).

**Comment by CerenB** [comment]
is there a way that we can be bids-valid, but save this config file under another name? 

I'm currently using `createJson(cfg, 'func')` but it would be nice to use in addition `createJson(cfg,cfg)` as well.

**Comment by Remi-Gau** [comment]
Yeah I can make it have a bids ISH name by default but make it possible to pass it another name
